### DISCUSSION TITLE
is it possible to have a reactive store in locals.store?

### DISCUSSION THREAD

**Discussion by ioucyf** [sveltejs/kit#11001] [Q&A]
I have a defined set of data, that does not update very often. For that reason, I decided to just store it as `data.json` file, and import it `hooks.server.ts` and assign it to `locals.data`.

if it ever needed an update, i'll just update the `data.json` file, and redeploy to vercel. So far, this works fine.

However, I thought, instead of the static `data.json` file, maybe I can have a cron job that checks for updates once a day, if any, it'll just update a reactive store that's assigned into locals so that it'll always have latest data; avoiding making lots of calls to api/database.

how can I do that?

I tried doing it, but the data always stales, and never updates, defeating the purpose of having a "reactive" store.

**Comment by teemingc** [comment]
Where does your cron job read the new data from?
